src/libfuncs/int.rs

```diff
@@ -1470,13 +1488,38 @@ mod test {
         for (value, target) in data {
             let result = executor.invoke_dynamic(&program.funcs[0].id, &[value.into()], None)?;
 
-            assert_eq!(result.builtin_stats.range_check, 1);
+            // If try_from(value) fails, means the value is out of range for T
+            match target {
+                Some(_) => {
+                    let range_size = T::max_value().into() - T::min_value().into() + BigInt::one();
+                    let rc_size = BigInt::from(1) << 128;
+                    if range_size < rc_size {
+                        assert_eq!(
+                            result.builtin_stats.range_check, 2,
+                            "Type: {}  Value: {}",
```

Comment:
To keep consistency:
```suggestion
                            "Difference in range_check count. Type: {}  Value: {}",
```